pkg/aspect/config/quotas.proto

```diff
@@ -32,7 +33,15 @@ message QuotasParams {
         // Map of quota_descriptor label name to attribute expression. At run time each
         // expression will be evaluated to determine the value provided to the aspect. The
         // result of evaluating the expression must match the ValueType of the label in the quota_descriptor.
-        map<string, string> labels = 3;
+        map<string, string> labels = 2;
```

Comment:
Proto field compatibility only matters when there are multiple binaries that depend on the proto definitions. Since no other binaries have been compiled that depend on our config APIs, we can change field numbers without breaking things. Even better, since our config is YAML, it doesn't care about field numbers so nothing breaks in our yaml ingestion either.

Once istioctl talks to our config API and has a dependency on our protos we can't shuffle these fields anymore.